components/Button/ButtonChildren/index.module.scss

```
@@ -0,0 +1,89 @@
+@import '/styles/color';
+@import '/styles/fonts';
+
+.standard-flex {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.add-column {
+  gap: 12px;
+
+  &.desktop {
+    height: 22px;
+  }
+
+  &.tablet {
+    height: 22px;
+  }
+
+  &.mobile {
+    height: 20px;
+  }

```

이 부분은 사실 조금 이해가 안되는데.. 사이즈에 맞게 클래스 네임을 각각 주시게끔 한거죠?!
근데 클래스네임을 주는거보다 미디어쿼리를 사용해서 기본 22px 해놓고 mobile 크기에만 20px이 되도록 하면 낫지 않을까.. 싶은데 어떻게 생각하세요?!
```suggestion
.add-column {
  gap: 12px;
  height: 22px;
  
  @include mobile{
    height:20px;
  }
```

믹스인 만들어둔게 있어서 이렇게 사용하면 코드가 많이 줄어들거같거든요! 
만약 이렇게 하신 이유가 있다면 설명해주시면 감사하겠습니다~!

---

혹시 테스트를 위해 설정하신 부분인가요..?!